apps/web/src/modules/dashboard/DashboardLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Flex, Text } from '@zoralabs/zord'
+import React, { ReactNode } from 'react'
+
+import { Meta } from 'src/components/Meta'
+
+export const DashboardLayout = ({
+  auctionCards,
+  daoProposals,
+}: {
+  auctionCards: ReactNode
+  daoProposals?: ReactNode
+}) => {
+  return (
+    <DashPage>
```

Comment:
Not sure if we need this to be a separate component if its not being reused anywhere else, would be easier to read as a single component

Reply:
Would definitely agree if it weren't being reused twice more in Dashboard.tsx